src/test/java/uk/gov/hmcts/dts/fact/model/CourtReferenceTest.java

```diff
@@ -23,6 +23,7 @@ static void setUp() {
         courtEntity.setSlug("name-slug");
         courtEntity.setNameCy("Name in Welsh");
         courtEntity.setUpdatedAt(currentTime);
+        courtEntity.setDisplayed(true);
     }
 
     @ParameterizedTest
```

Comment:
You set it to be true in the setup method. Do you mean to test it somewhere later?